roles/confluent.control_center/tasks/health_check.yml

```diff
@@ -28,3 +28,8 @@
     group: "{{control_center_group}}"
   # Only one health check runs, only one will have a 'failed' field. For skipped checks, defaulting 'failed' to False
   when: result1.failed|default(False)
+
+- name: Fail Provisioning
+  fail:
```

Comment:
@erikgb any idea whats wrong with this task linting wise?
https://ansible-lint.readthedocs.io/en/latest/default_rules.html#ignore-errors

Am i not allowed to use the fail module?

Reply:
This is the error message:

> 35:38     error    no new line character at the end of file  (new-line-at-end-of-file)

At least now. 😄

Reply:
Oh duh! I was reading the ansible-lint logs. Thanks